force-app/main/default/lwc/formattedRecipeDisplay/__tests__/data/getRecipeCode.json

```diff
@@ -1,6 +1,6 @@
     "groupName": "Data Recipes",
     "lengthWithoutComments": 3265,
```

Comment:
Not sure whether this is important but should you update the length following your changes?